src/test/java/seedu/address/model/loan/LoanTimeTest.java

```diff
@@ -43,19 +44,19 @@ public void constructorInputCheckCurrentDate() {
         // between the next two statements. Therefore, we'll also have the current
         // date after the LoanTime construction. The constructed LoanTime must be
         // equal to one of them.
-        String currentDateBeforeLoanTimeCreation = LocalDate.now().format(DateTimeFormatter.ofPattern("uuuu-MM-dd"));
+        String currentDateBeforeLoanTimeCreation = LocalDate.now().format(EXPECTED_DATE_FORMAT);
         LoanTime loanTime1 = new LoanTime("00:25");
         LoanTime loanTime2 = new LoanTime("21:03");
-        String currentDateAfterLoanTimeCreation = LocalDate.now().format(DateTimeFormatter.ofPattern("uuuu-MM-dd"));
+        String currentDateAfterLoanTimeCreation = LocalDate.now().format(EXPECTED_DATE_FORMAT);
 
         String loanTime1ToString = loanTime1.toString();
         String loanTime2ToString = loanTime2.toString();
 
         assertTrue(loanTime1ToString.equals(currentDateBeforeLoanTimeCreation + ", 00:25")
-                || loanTime1ToString.equals(currentDateAfterLoanTimeCreation + ", 00:25"));
+                || loanTime1ToString.equals(currentDateAfterLoanTimeCreation + " 00:25"));
 
         assertTrue(loanTime2ToString.equals(currentDateBeforeLoanTimeCreation + ", 21:03")
```

Comment:
There are some commas here to delete as well, on lines 55 and 58. Apart from that, everything else is good.

Reply:
Ah man. Good catch. I'll fix it real quick.